src/Block/Dashboard/MonitorBlockService.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the ekino/sonata project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\HelpersBundle\Block\Dashboard;
+
+use Liip\MonitorBundle\Helper\PathHelper;
+use Sonata\BlockBundle\Block\BlockContextInterface;
+use Sonata\BlockBundle\Block\Service\AbstractAdminBlockService;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class MonitorBlockService.
+ *
+ * @author Quentin Belot <quentin.belot@ekino.com>
+ */
+class MonitorBlockService extends AbstractAdminBlockService
+{
+    /**
+     * @var PathHelper
+     */
+    private $pathHelper;
+
+    /**
+     * @var string
+     */
+    private $liipMonitorDefaultGroup;
+
+    /**
+     * MonitorBlockService constructor.
+     *
+     * @param string          $name
+     * @param EngineInterface $templating
+     * @param PathHelper      $pathHelper
+     * @param string          $liipMonitorDefaultGroup
+     */
+    public function __construct(
+        string $name,
+        EngineInterface $templating,
+        PathHelper $pathHelper,
+        string $liipMonitorDefaultGroup
+    ) {
+        parent::__construct($name, $templating);
+
+        $this->pathHelper              = $pathHelper;
+        $this->liipMonitorDefaultGroup = $liipMonitorDefaultGroup;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureSettings(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'template' => '@SonataHelpers/Block/dashboard/monitor.html.twig',
+            'ttl'      => 300,
```

Comment:
useless to me because it will not be read as it is included through sonata admin configuration. BTW we are expected real time data for those checks (no cache)